src/pudl/transform/ferc1.py

```diff
@@ -3018,6 +3024,21 @@ def targeted_drop_duplicates_dbf(self, df: pd.DataFrame) -> pd.DataFrame:
         )
         return deduped
 
+    # def add_plant_function(self, df: pd.DataFrame) -> pd.DataFrame: # TO DO: Figure out where to add to not throw off calcs.
+    #     """Label factoids with plant function."""
```

Comment:
@cmgosnell Here's a first attempt at plant function adding that immediately caused calc trouble, if you're interested.